shared-module/displayio/Display.c

```diff
@@ -82,7 +83,13 @@ void common_hal_displayio_display_construct(displayio_display_obj_t* self,
         data_size &= ~DELAY;
         uint8_t *data = cmd + 2;
         self->send(self->bus, true, cmd, 1);
-        self->send(self->bus, false, data, data_size);
+        if (self->data_as_commands) {
+            for (uint32_t j=0; j < data_size; j++) {
```

Comment:
Why is this in a for loop rather that one call for all of the data? Is it because CS needs to be toggled?

Reply:
(Please add the explanation as a comment if it is needed.) Thanks!

Reply:
Yes, exactly.

Reply:
Done.